javascript/src/main/pages/AnalyzeMessageData/CountMessagesByUser.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import BootstrapTable from 'react-bootstrap-table-next';
+import useSWR from "swr";
+import {useAuth0} from "@auth0/auth0-react";
+import {fetchWithToken} from "main/utils/fetch";
+import aggregateUserMessageCount from "main/utils/aggregateUserMessageCount";
+
+const CountMessagesByUser = () => {
+    const columns = [{
+        dataField: 'name',
+        text: 'user'
+    } , {
+        dataField: "count",
+        text: 'Message Count'
+    }];
+    const { getAccessTokenSilently: getToken } = useAuth0();
+    const { data: slackUsers } = useSWR(["/api/slackUsers", getToken], fetchWithToken);
+    const { data: messages } = useSWR(["/api/members/messages/allmessages", getToken], fetchWithToken);
+
+    const userMessageCount = messages && slackUsers ? aggregateUserMessageCount(messages, slackUsers) : []
+    
+    return (
+        <div>
+        <h1>Count Messages By User</h1>
```

Comment:
Tab indent for line 24 and 25 so that it is clear that they are nested under the div